src/test/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtilTest.java

```diff
@@ -382,6 +392,84 @@ public void testEscapeAllControlChars() {
                 JavadocUtil.escapeAllControlChars("1\\r2\\n3\\t"), "invalid result");
     }
 
+    @Test
+    public void testGetContentOfInlineCustomTagWithNestedTags() throws Exception {
```

Comment:
All such tests shy be removed.
All coverage should be done by Input files through Check.

Reply:
Done.